components/dashboard/src/projects/Project.tsx

```diff
@@ -80,36 +78,13 @@ export default function () {
         entries.push({
             title: "Trigger Prebuild",
```

Comment:
**issue**: Posting this here as it could be related. When loading a team page with the projects list, I get no prebuilds information and see the following error in the console.

```js
Uncaught (in promise) Error: Request findPrebuilds failed with message: Not Found
```

Reply:
It was a bug indeed. Resolved and deployed ✔️